src/libs/metadata.c

```diff
@@ -126,7 +127,31 @@ static void _update(dt_lib_module_t *self)
 {
   dt_lib_cancel_postponed_update(self);
   dt_lib_metadata_t *d = (dt_lib_metadata_t *)self->data;
+
   d->imgsel = dt_control_get_mouse_over_id();
```

Comment:
It is not yours (and probably mine :) ), but I think imgsel is not used anymore. Could be removed (3 occurrences).

Reply:
will take care too